README.md

```diff
@@ -1,18 +1,110 @@
+Search the IBM jobs and careers website for additional quantum computing opportunities not listed below:  
+http://ibm.biz/quantum-careers
 
-**Quantum Developer Intern — Summer 2021:** Developer interns will have the opportunity to focus on kernel, algorithm, or model development. Areas of focus could include quantum machine learning, quantum biology and health informatics, quantum algorithms for chemistry, quantum simulations, quantum optimization, cloud microservices, and cloud API (among other areas). PhD student developers can apply to the [PhD Quantum Developer Summer Intern](https://careers.ibm.com/ShowJob/Id/962210/PhD-Quantum-Developer-Summer-Intern-2021/?lang=en) job requisite. Master’s student developers can apply to the [Graduate Quantum Developer Summer Intern](https://careers.ibm.com/ShowJob/Id/962202/Graduate-Quantum-Developer-Summer-Intern-2021/?lang=en) job requisite. Undergraduate student developers can apply to the [Undergraduate Quantum Developer Summer Intern](https://careers.ibm.com/ShowJob/Id/962204/Undergraduate-Quantum-Developer-Summer-Intern-2021/?lang=en) job requisite. Developer interns will work with either the IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA) or IBM Research — Almaden (San Jose, California, USA).
+## Internships
 
-**Quantum Engineer Intern — Summer 2021:** Engineering interns will have the opportunity to focus on hardware of software engineering. Areas of engineering could include superconducting qubits, quantum software architecture, cryoelectronic circuit design, quantum microwave engineering, and quantum FPGA engineering (among other areas). PhD student engineers can apply to the [PhD Quantum Engineering](https://careers.ibm.com/ShowJob/Id/962213/PhD-Quantum-Engineering-Summer-Intern-2021/?lang=en) job requisite. Master’s student engineers can apply to the [Graduate Quantum Engineering Summer Intern](https://careers.ibm.com/ShowJob/Id/962214/Graduate-Quantum-Engineering-Summer-Intern-2021/?lang=en) job requisite. Undergraduate student engineers can apply to the [Undergraduate Quantum Engineering Summer Intern](https://careers.ibm.com/ShowJob/Id/962215/Undergraduate-Quantum-Engineering-Summer-Intern-2021/?lang=en) job requisite. Engineer interns will work with the IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA) or the IBM Systems labs in Rochester, MN or Poughkeepsie, NY.
+Read *Apply for a Summer 2021 IBM Quantum Internship* on the IBM Research Blog:  
+https://ibm.co/quantuminternships
 
-Search the [IBM jobs and careers website for more quantum computing opportunities](https://careers.ibm.com/ListJobs/All/Search/jobtitle/quantum/) at IBM.
+_For IBM Quantum internships in the United States, please apply by Monday, November 2, 2020 in order to receive the best chance at consideration._
 
```

Comment:
Should we make a statement about international opportunities still being forthcoming?